packages/components/organisms/f-content-cards/src/components/cards/PromotionCardOne.vue

```diff
@@ -0,0 +1,53 @@
+<template>
+    <content-card-container
+        #default="{
```

Comment:
Feeling rusty today, what does this syntax do again?

Reply:
I think it's reading the values out of the `content-card-container` component to be used in this component, right?

Reply:
yep reading values out of the `content-card-container` as they are slot props.